src/locales/en.json

```diff
@@ -214,6 +214,10 @@
     "saveAll": {
       "text": "Save All",
       "tooltip": "Download a JSON file of the institutions"
+    },
+    "delete": {
+      "text": "Delete Institution",
+      "tooltip": "Delete Temporary Institution"
```

Comment:
Replace with "Remove from the list" so it can apply to institutions and panels

Reply:
updated